### PR TITLE
Activation of candidates url and menu on due date

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: app/templates/navbar.html [hunk 1]
```diff
@@ -14,7 +14,10 @@
         <div id="navbar-collapse-1" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
                 {% for child in children %}
-                <li><a href="{{ child.attr.redirect_url|default:child.get_absolute_url }}">{{ child.get_menu_title }}</a></li>
+                    <!-- Delete this if statement on the due date -->
+                    {% if child.get_menu_title != "Kandidaten und Kandidatinnen" and child.get_menu_title != "Candidats et candidates" and child.get_menu_title != "Candidati e candidate" %}
```
Comment (winged) [inline app/templates/navbar.html]: If you can't do it from the CMS, maybe it'd be better to match the URL instead

File: app/core/urls.py [hunk 2]
```diff
@@ -16,10 +16,11 @@
         r'^i18n/',
         include('django.conf.urls.i18n')
     ),
-    url(r'^candidates/$',
-        views.candidates_view,
-        name='candidates'
-    ),
+    # activate this url on the due date
```
Comment (winged) [inline app/core/urls.py]: Why remove this? Should be enough to just not show the menu

File: app/templates/navbar.html (hunk 1 above)
Comment (JohnL17) [reply app/templates/navbar.html]: I have changed this.

File: app/core/urls.py (hunk 2 above)
Comment (JohnL17) [reply app/core/urls.py]: The url is enabled again.

**Pull request merged**